rotkehlchen/chain/ethereum/modules/defisaver/decoder.py

```diff
@@ -64,10 +141,51 @@ def _decode_substorage_action(self, context: DecoderContext) -> DecodingOutput:
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_flashloan_events(self, context: DecoderContext) -> DecodingOutput:
+        """
+        Decodes DeFi Saver events from the given context.
+        """
```

Comment:
Fits in a line and be specific

```suggestion
    """Decode flashloan events"""
```